src/styles/styles/_card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+  
+  &__lable {
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+  }
+  
+  &__code-product {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    padding-top: 4px;
+    color: $color-secondary;
+  }
+  &__price-position {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 24px;
+    padding-bottom: 16px;
+  }
+  
+  &__price-lable {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+  
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $mainColor;
+    align-self: flex-end;
+  }
+  
+  &__button {
+    margin: 0 auto;
+    display: flex;
+    color: $whiteAccent;
+    width: 166px;
+    height: 40px;
+    background-color: $blueAccent;
+    border: none;
+    border-radius: 5px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+  }
+
+  &__button:hover {
+    border: 1px solid $blueAccent;
+    background-color: $whiteAccent;
+    border-radius: 5px;
+    color: $blueAccent;
+  }
+}
+
+.reviews__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 16px;
+  gap: 17px;
+}
+
+.reviews__text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
now you would need to create a separated file for it, but we will move it in `.card` block after html rework